imager/profiles/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.conf import settings
+import datetime
+
+
+class ImagerProfile(models.Model):
+
+    user = models.OneToOneField(settings.AUTH_USER_MODEL)
+    picture = models.ImageField(default='nophoto.jpg')
+    phone = models.CharField(max_length=20, default='No Phone')
+    birthday = models.DateField(default=datetime.date.today)
+
+    name_priv = models.BooleanField(default=True)
+    email_priv = models.BooleanField(default=True)
+    picture_priv = models.BooleanField(default=True)
+    phone_priv = models.BooleanField(default=True)
+    birthday_priv = models.BooleanField(default=True)
+
+    def __str__(self):
+        return self.user.username
+
+    def is_active(self):
+        return self.user.is_active
+
+    @classmethod
+    def active(cls):
+        active_users = []
+        profiles = ImagerProfile.objects.all()
+        for prof in profiles:
+            if prof.user.is_active is True:
+                active_users.append(prof)
+        return active_users
```

Comment:
classmethods on Django models should almost always be methods on a cutsom model manager.